lib/generate.js

```diff
@@ -35,36 +37,64 @@ function dedentFunc(func) {
     });
 }
 
+function tabulateString(string, indent, filter) {
```

Comment:
the filter argument is passed the id of the current line along with the line itself. `id` will return 0 for id 0, thus preventing the indentation of the first line.